usr/share/rear/lib/layout-functions.sh

```diff
@@ -530,7 +530,12 @@ get_partition_start() {
 
 # Get the type of a layout component
 get_component_type() {
```

Comment:
It would help me (and maybe others) to have typical output in a comment to better understand what the code actually does.

Reply:
See [290a2b3](https://github.com/rear/rear/pull/3403/commits/290a2b3aec0f99f779ad0bdcb5a1ee41b459eb33) with clarification